### DISCUSSION TITLE
rinna's Japanese GPT model support

### DISCUSSION THREAD

**Discussion by katsu560** [ggml-org/ggml#32] [General]
Thanks @ggerganov for your sharing.

I want to use GPT on my local pc.
rinna Co.,Ltd is Japanese AI company. rinna provides some GPT models on huggingface.
https://huggingface.co/rinna/japanese-gpt-1b
https://huggingface.co/rinna/japanese-gpt2-xsmall
https://huggingface.co/rinna/japanese-gpt2-small
https://huggingface.co/rinna/japanese-gpt2-medium
I'd like to use these models with ggml gpt.
But, I can't convert these models for ggml.
I think these models are pytorch bin model and tensorflow H5 weight.

Could you convert and support these models ?

move to issue #33
could you delete this?

**Comment by ocordeiro** [comment]
You can use [examples/gpt-2/convert-h5-to-ggml.py](https://github.com/ggerganov/ggml/discussions/32#diff-2583aed0552c1e8a1c5809e0421e31db27a4053f9757f853c6c71ba439aa87d5)  to convert these models:

https://github.com/ggerganov/ggml/pull/35
